source/_integrations/envertech_evt800.markdown

```diff
@@ -0,0 +1,108 @@
+---
+title: Envertech EVT800
+description: Instructions on how to connect your Envertech EVT800 device to Home Assistant.
+ha_iot_class: Local Polling
+ha_category:
+  - Energy
+ha_config_flow: true
+ha_release: 2025.8
+ha_codeowners:
+   - '@daniel-bergmann-00'
+ha_domain: envertech_evt800
+ha_integration_type: integration
+---
+
+The Envertech EVT800 enables {% term integration %} with [Envertech EVT800](https://www.envertec.com/products/microinverter/66.html/) solar inverters in Home Assistant.
+
+## Supported Devices
+
+The following devices are known to be supported by this integration:
+
+- Envertech EVT800
+
+## Prerequisites
+
+Before setting up the integration, ensure the following:
+
+1. Your EVT800 is powered by solar (it does not power on without sunlight).
+2. The EVT800 is connected to your local network via Wi-Fi.
+3. Determine the **IP address** of the EVT800:
+   - Check your router`s device list or use a network scanning tool.
+4. Open a web browser and navigate to:
+   `http://<YOUR_EVT800_IP_ADDRESS>`
+   - Default username: `admin`
+   - Default password: `admin`
+5. Go to **Other settings** and set the **Protocol** to **TCP Server**.
+
+{% include integrations/config_flow.md %}
+
+## Configuration Variables
```

Comment:
you could use a configuration_basic block for this section. As shown in [this example](https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L57-L62).

<img width="1167" height="140" alt="image" src="https://github.com/user-attachments/assets/3acf7ac7-9e4b-4637-bebb-65cae64930d3" />